src/OpenLoco/src/World/StationManager.cpp

```diff
@@ -527,6 +528,20 @@ namespace OpenLoco::StationManager
         return deliverCargoToStations(foundStations, cargoType, cargoQty);
     }
 
+    // 0x0048FEF4
+    bool exceedsStationSize(Station& station, World::Pos3 pos)
+    {
+        if (Config::get().disableStationSizeLimit)
+            return false;
+
+        auto centreTile = World::Pos2(pos.x + World::kTileSize / 2, pos.y + World::kTileSize / 2);
+        auto stationCentre = World::Pos2(station.x, station.y);
+        auto distance = Math::Vector::manhattanDistance2D(centreTile, stationCentre);
+
+        const auto stationDistanceLimit = 8 * World::kTileSize;
```

Comment:
should make this a kStationDistanceLimit and move it out of the function.

Reply:
What would be a good place for the constant? S5/Limits.h?

Reply:
hmm probably fine to just leave it in this file.